resources/js/app.js

```diff
@@ -1 +1,15 @@
 import './bootstrap';
+        document.getElementsByClassName("eye-icon").addEventListener("click", function() {
```

Comment:
Whats this js for, if you wrote the entire script in the controller as well

Reply:
its to enable the user to view their password when changing it

Reply:
i must have forgotten to delete it.........i was first writing the js their so that it doesnt have errors